mesonbuild/mtest.py

```diff
@@ -697,10 +697,10 @@ def preexec_fn() -> None:
 
         extra_cmd = []  # type: T.List[str]
         if self.test.protocol is TestProtocol.GTEST:
-            gtestname = '{}.xml'.format(self.test.name)
+            gtestname = '{}'.format(self.test.name)
             if self.test.workdir:
-                gtestname = '{}:{}'.format(self.test.workdir, self.test.name)
-            extra_cmd.append('--gtest_output=xml:{}'.format(gtestname))
+                gtestname = '{}/{}'.format(self.test.workdir, self.test.name)
```

Comment:
This should use `os.path.join`, instead of hard coding `/`.

Reply:
Fixed